.github/workflows/major-version-upgrade.yml

```diff
@@ -6,8 +6,8 @@ jobs:
     env:
       OLD_INSTALL_DIR: psql_source
       NEW_INSTALL_DIR: psql_target
-      ENGINE_BRANCH_FROM: BABEL_2_X_DEV__PG_14_X
```

Comment:
Let's also upgrade version-branch-template for 19.4 [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_3_STABLE/.github/template/version-branch-template.yml#L34-L36)

Reply:
Done!